helloworld-simple-project/README.md

```diff
@@ -0,0 +1,41 @@
+# hello-quarkus  Project
+
+This project uses Quarkus, the Supersonic Subatomic Java Framework.
+
+If you want to learn more about Quarkus, please visit its website: https://quarkus.io/ .
+
+
+## Creating a Serverless Function
+
+For Quarkus,
+                  kn func create -l quarkus -t http function-name
```

Comment:
I think you can use code blocks for your readme. Also you can change your readme markdown file to an adoc file. That might be a good adoc practice for you.

here is our ref guide for adoc: https://github.com/RedHatTraining/skeleton/blob/main/documentation/ref_guide.adoc

Github might parse some things in a different way but most of them are valid for github too.

Reply:
Sure will check on this